src/aria/widgets/form/MultiSelect.js

```diff
@@ -85,6 +86,18 @@ module.exports = Aria.classDefinition({
          * @type aria.widgets.form.list.List
          */
         this._dropDownList = null;
+        var isWaiAria = cfg.waiAria;
+        var iconTooltip = cfg.iconTooltip ? ' title="' + ariaUtilsString.escapeForHTML(cfg.iconTooltip) + '"' : '';
+        var tabIndex = isWaiAria ? (cfg.tabIndex != null ? this._calculateTabIndex() : "0") : "-1";
+        this._iconsAttributes = {
+            "dropdown": 'tabindex="' + tabIndex + '"' + iconTooltip
+        };
+
+        if (isWaiAria) {
+            var waiIconLabel = cfg.waiIconLabel;
+            this._iconsAttributes.dropdown += ' role="button" aria-expanded="false" aria-haspopup="true"  ' +
+               (waiIconLabel ? 'aria-label="' + waiIconLabel + '" ' : "");
+        }
```

Comment:
Why do you need this?
Isn't the change in `DropDownTextInput` (adding role="button") sufficient?